requirements.txt

```diff
@@ -1 +1 @@
-ansible==2.1.0
+ansible==2.2.0
```

Comment:
I'd change this to `ansible>2.1` that way we pick up any bug fixes or security fixes that may land in 2.2